src/resmom/linux/mom_mach.c

```diff
@@ -222,20 +225,21 @@ pbs_plinks	*Proc_lnks = NULL;	/* process links table head */
 static time_t	sampletime_ceil;
 static time_t	sampletime_floor;
 
+#if	MOM_CPUSET
 typedef int		pidcachetype_t;		/* type of allocatable unit */
 static pidcachetype_t	*pidcache_arena;	/* cache storage area */
 static unsigned int 	pidcache_bitsper = sizeof(pidcachetype_t) * NBBY;
 static int		pidcache_check(pid_t, pidcachetype_t *);
 static pidcachetype_t	*pidcache_create(void);
+static int		pidcache_reset(pidcachetype_t *);
 static void		pidcache_destroy(void);
-static int		pidcache_eligible(proc_stat_t *, char *, char *, unsigned long long);
 static pidcachetype_t *	pidcache_getarena(void);
```

Comment:
I am not sure why it is showing like thin in diff mode. Please have a look with "view the whole file" and you should see the indentation right.